checkout-ui-custom/src/partials/Deliver/ExtraFieldsContainer.js

```diff
@@ -2,7 +2,7 @@ import RICAForm from './RICAForm';
 import TVLicenseForm from './TVLicenseForm';
 
 const ExtraFieldsContainer = ({ hasTV, hasSim }) => {
-  const showTVLicenseForm = `
+  const showTVLicenseForm = /* html */`
```

Comment:
But why? Legit interested

Reply:
Never mind, I see why 😆

Reply:
there is a really nice plugin for vs code that that will highlight string templates as html as below

![image](https://github.com/user-attachments/assets/66a3050e-682c-4898-82a9-f5f59cae2ed8)

because the whole repo is string literals, this is a nice way to dev

Reply:
I just got the plugin, is nice